mirascope/cli/commands.py

```diff
@@ -267,10 +269,83 @@ def use(
     print(f"Using {revision_file_path}")
 
 
+@app.command(help="Remove a prompt")
+def remove(
+    prompt_file_name: str = Argument(
+        help="Prompt file to remove",
+        autocompletion=_prompts_directory_files,
+        parser=_parse_prompt_file_name,
+        default="",
+    ),
+    version: str = Argument(
+        help="Version of prompt to use",
+    ),
+):
+    """Removes the version from the versions directory
+
+    All versions with prev_revision_id matching the deleted version are detached.
+
+    Args:
+        prompt_file_name: The name of the prompt file to remove.
+        version: The version of the prompt to remove
+
+    Raises:
+        FileNotFoundError: If the file is not found in the specified prompts or
+        versions directory.
+    """
+    mirascope_settings = get_user_mirascope_settings()
+    version_directory_path = mirascope_settings.versions_location
+    version_file_name = mirascope_settings.version_file_name
+    prompt_versions_directory = os.path.join(version_directory_path, prompt_file_name)
+    version_file_path = os.path.join(prompt_versions_directory, version_file_name)
+
+    revisions = get_prompt_versions(version_file_path)
+    if revisions.current_revision == version:
+        print(
+            f"Prompt {prompt_file_name} {version} is currently being used. "
+            "Please switch to another version first."
+        )
+        return
+
+    revision_file_path = find_prompt_path(prompt_versions_directory, version)
+    if not revision_file_path:
+        raise FileNotFoundError(
+            f"Prompt version {version} not found in {prompt_versions_directory}"
+        )
+    # TODO: Implement rollback in case of failure
```

Comment:
should we make this a good first issue instead of a TODO?

Reply:
we can keep the todo and make it an issue